velox/docs/functions/regexp.rst

```diff
@@ -33,3 +33,19 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     the entire string by anchoring the pattern using ``^`` and ``$``::
 
         SELECT regexp_like('1a 2b 14m', '\d+b'); -- true
+
+.. function:: like(string, pattern) -> varchar
+              like(string, pattern, escape) -> varchar
```

Comment:
I thought "like" should return "bool", no?

Reply:
I misread the PrestoDB docs to interpret "like" as a vector function that returned the original column values. But a boolean operator does seem more preferable. Have changed this.